storage-access.bs

```diff
@@ -293,6 +307,42 @@ ISSUE: Write this section.
 
 ISSUE: Write this section.
 
+<h2 id="automation">Automation</h2>
+
+For the purposes of user-agent automation and application testing, this document defines the following [=extension command=] for the [[WebDriver]] specification.
+
+<h3 id="set-storage-access-command">Set Storage Access</h3>
+
+<table>
+    <tbody>
+    <tr>
+        <th>HTTP Method</th>
+        <th>URI Template</th>
+    </tr>
+    <tr>
+        <td>POST</td>
+        <td>/session/{session id}/storageaccess</td>
+    </tr>
+    </tbody>
+</table>
+
+The <dfn export>Set Storage Access</dfn> [=extension command=] modifies the storage access policy for the [=current browsing context=].
+
+The [=remote end steps=] are:
+
+1. If |parameters| is not a JSON [=Object=], return a [=WebDriver error=] with [=WebDriver error code=] [=invalid argument=].
+1. Let |blocked| be the result of [=getting a property=] from |parameters| named `blocked`.
+1. If |blocked| is not a [=boolean=] return a [=WebDriver error=] with [=WebDriver error code=] [=invalid argument=].
+1. Let |doc| be the [=current browsing context=]'s [=active document=].
+1. Let |key| be the result of [=generate a partitioned storage key|generating a partitioned storage key=] from |doc|.
+1. If |key| is failure, return a [=WebDriver error=] with [=WebDriver error code=] [=unknown error=].
```

Comment:
Nit: I prefer "unsupported operation" over "unknown error". This is only likely to fail if the remote end (say, WebKit) doesn't support the operation.

Reply:
Thanks. I was on the fence about this myself. Switching to "unsupported operation".